Source/SwiftLintFramework/Rules/ModifierOrderRule.swift

```diff
@@ -183,9 +183,12 @@ public struct ModifierOrderRule: ASTRule, OptInRule, ConfigurationProviderRule {
         let filteredPreferedOrderOfModifiers = preferredOrderOfModifiers.filter(modifierGroupsInDeclaration.contains)
         for (index, preferredGroup) in filteredPreferedOrderOfModifiers.enumerated()
             where preferredGroup != modifierGroupsInDeclaration[index] {
+                let failureReason = "\(preferredGroup) modifier should be before " +
```

Comment:
this is already an improvement, but would it be possible to print the exact modifiers instead of their groups?

If it's not hard, we should do that. Otherwise we can merge as it is and then figure out later.

Reply:
I have figured out a not so complex way to get the  exact modifiers instead of the groups. I will push my changes soon to this PR.